Simplenote/Resources/AppStoreStrings.pot

```diff
@@ -63,11 +64,13 @@ msgstr ""
 #. translators: Keywords used in the App Store search engine to find the app.
 #. Delimit with an English comma between each keyword. Limit to 100 characters including spaces and commas.
 msgctxt "app_store_keywords"
-msgid "notes,text,planner,journal,diary,sync,to-do,cloud,notebook,simple,notepad,tag,todo,writing,memo"
+msgid "notes,text,planner,journal,diary,sync,to-do,cloud,notebook,simple,notepad,tag,todo,writing,memo
+"
 msgstr ""
 
-msgctxt "v4.41-whats-new"
+msgctxt "v4.42-whats-new"
 msgid ""
-"- Some bug fixes\n"
+"-   Added In app account deletion #1354\n"
+"\n"
```

Comment:
Just got a brainfart: Simplenote release notes don't go thru Editorial like other apps do, and that's why it's ok to update the pot right away here (as opposed to waiting for final notes drafted by Editorial), right?

Reply:
That's correct.